README.md

```diff
@@ -139,13 +116,38 @@ To change the output format to tab separated values (TSV), add `-o tsv` to the c
 
 All debug, informational, warning, and error information is sent to `stderr`. This is so it can be easily separated from the command output. You can decide what level of logging you'd like by using the `-l` option. The valid levels of logging messages are `debug`, `info`, `warn`, `error`, and `none`.
 
+## Projects
+
 ### Multiple Twilio accounts/projects
 
-To store credentials for multiple projects, you can use a shorthand "project id" which is just an easy to remember, short string to identify the project. (If you've used `git` before, it's like the name you assign to a remote like "origin".)
+When you run `twilio login`, it stores your credentials and associates them with the provided project ID. The first project added will default to being the "active" project. The active project is used for all subsequent commands.
+
+To add additional projects, run `twilio login` again but provide a different project ID (like, `my_other_proj`). Then, when you run subsequent commands, just include `-p my_other_proj` in the command (e.g. `twilio incoming-phone-number:list -p my_other_proj`).
```

Comment:
I think instead of asking the user to do a `twilio login` for adding additional projects, we can use `twilio project:add`. In the context of adding a new project, `twilio login` makes less sense to me. I can see where you can reason that you're logging into a new project, but if you are already in a project the normal workflow would be to logout first.

Reply:
I think throwing in `project:add` is confusing since we're using `login` everywhere?

@aroach Thoughts?

Reply:
A few lines later we introduce `project:use` and `project:list`, so I was thinking for this case `project:add` made sense.

Reply:
how about we put in a parenthetical. "When you run `twilio login` (an alias for twilio project:add), it stores..."

Reply:
Updated.